frontend/package.json

```diff
@@ -37,7 +37,7 @@
     "typescript": "^4.8.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
can you remove this addition? it's compatible only with React version 18+ and I do not think all the developers on that version. It would be better to ask all the developers to upgrade in a separate dedicated Pull Request